hack/lib/util/golang.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# This library holds golang related utility functions.
+
+# Ensure the go tool exists and is a viable version.
+os::golang::verify_go_version() {
+  if [[ -z "$(which go)" ]]; then
+    cat <<EOF
```

Comment:
Yuck, the current implementation of `os::log::info` is not the best and it doesn't honor newlines, so I think you'll have to do:
```sh
os::log::info "first line"
os::log::info "second line"
os::log::info "third line"
```

That actually might look better too since it will all be formatted the same and line up and stuff